### PR TITLE
updated the readme file to add some important links

### PR DESCRIPTION
**Pull request by eram23**

### CODE REVIEW

File: contributors.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [
     {
-        name: "Micha",
-        linkedIn: "https://linkedin.com/in/michaella-rodriguez",
-        gitHub: "https://github.com/michaella23",
+        name: "Erik",
```
Comment (michaella23) [inline contributors.js:3]: Hey @eram23, you will need to put your info inside of a separate object, after mine. Kind of like this -
```[
       { (me)
        name: "",
        linkedIn: "",
        gitHub: "",
        avatar: ""
      },
      { (you)
             name: "",
       linkedIn: "",
       gitHub: "",
       avatar: ""
       },
     ]

File: README.md [hunk 2]
```diff
@@ -12,3 +12,8 @@ Here is some important information that should be included in a README for an op
 - [ ] link to the open source license
 - [ ] link to community Discord, Slack group, or GitHub Discussions
 
+## Helpful links:
+[Contributing md](https://github.com/eram23/progress-tracker.git)
+[Code of conduct](https://github.com/scrimba/community/blob/master/CODE-OF-CONDUCT.md)
```
Comment (michaella23) [inline README.md:17]: We don't have a Code Of Conduct in this repository. There is a separate issue for that. Once it is created, we can add a link here.

File: README.md [hunk 3]
```diff
@@ -12,3 +12,8 @@ Here is some important information that should be included in a README for an op
 - [ ] link to the open source license
 - [ ] link to community Discord, Slack group, or GitHub Discussions
 
+## Helpful links:
+[Contributing md](https://github.com/eram23/progress-tracker.git)
```
Comment (michaella23) [inline README.md:16]: Doublecheck this link please.

**Review by michaella23 — CHANGES_REQUESTED** [review]
Hey Eric! I'm so glad you were able to create your first pull request. You'll need to make a few changes. Then you can add / commit / push again and comment in this PR that you have made the changes. Thank you!

File: contributors.js (hunk 1 above)
Comment (eram23) [reply contributors.js:3]: Understood, will update it now thanks.

File: README.md (hunk 2 above)
Comment (eram23) [reply README.md]: Gotcha, will leave the URL empty for now then if that is ok

Comment (michaella23) [reply README.md]: Yeah, that's fine!

**Review by michaella23 — APPROVED** [review]
Thanks for making these changes! Merging your PR! 

As you continue to contribute, you'll want to be sure and do the following before submitting another pull request:

1. Go to your forked repository on GitHub.
2. Click the "Sync fork" button.
3. Click the "Update branch" button.

Go to your main branch (in VS Code / the terminal) and pull changes. 
In the terminal, type `git switch main`, then `git pull`. 

You'll repeat the process in the future - request to work on an issue, create a feature branch, stage, commit, push, etc.!

**Pull request merged**